test/e2e_node/remote/gce/gce_runner.go

```diff
@@ -768,12 +768,15 @@ func (g *GCERunner) updateKernelArguments(instance *compute.Instance, image stri
 }
 
 func (g *GCERunner) machineType(machine string) string {
+	var ret string
 	if machine == "" && *instanceType != "" {
-		machine = *instanceType
+		ret = *instanceType
+	} else if machine != "" {
```

Comment:
this is OK. It may be better to have a big if statement when `machine` is empty with sub-if statement. The logic as written will work, just easier to make a mistake

Reply:
I'll revisit this after 1.28 is released